_posts/2020-02-27-react-conf-2020.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "ReactConf 2020"
+date:   2020-03-02
+tags: React Sydney
+author: Natalie Laing
+image: /img/blog/reactConf/react.png
+---
+
+## Introduction 
+React Conf AU is the first React conference in the southern hemisphere and it didn't disappoint. The quality of speakers was spot on, meet the faces behind the projects that make React what it is today.
```

Comment:
You may want to give an intro as to what this is.. at the moment seems like a blow by blow account of ReactConf or your personal highlights? Or?

Reply:
ok, more my highlights. Will re write.